src/mono/sample/wasm/browser-nextjs/Wasm.Browser.NextJs.Sample.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <WasmMainJSPath>package.json</WasmMainJSPath>
```

Comment:
I know it is a placeholder in this case. Shouldn't we do something with it here?

Reply:
https://github.com/dotnet/runtime/issues/63465